<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CannabusControl</class>
 <widget class="QMainWindow" name="CannabusControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CannabusControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="busStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>515</y>
      <width>781</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>480</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>760</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>Time       F-Code      Slave Address      Message Type      DLC      Data </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>760</width>
       <height>420</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionClearLog"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuCalls"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionClearLog"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../images/cannabus_control.qrc">
     <normaloff>:/connect.png</normaloff>:/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to the CAN Device</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="../images/cannabus_control.qrc">
     <normaloff>:/disconnect.png</normaloff>:/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from the CAN Device</string>
   </property>
  </action>
  <action name="actionClearLog">
   <property name="icon">
    <iconset resource="../images/cannabus_control.qrc">
     <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear Log</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../images/cannabus_control.qrc">
     <normaloff>:/application-exit.png</normaloff>:/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images/cannabus_control.qrc"/>
 </resources>
 <connections/>
</ui>
